tests/vmcheck/overlay.sh

```diff
@@ -59,12 +59,16 @@ fi
 cd /ostree/repo/tmp
 rm vmcheck -rf
 ostree checkout $commit vmcheck --fsync=0
+# https://github.com/projectatomic/rpm-ostree/pull/693
+rm -f vmcheck/usr/etc/{.pwd.lock,passwd-,group-,shadow-,gshadow-,subuid-,subgid-}
 rsync -rlv /var/roothome/sync/insttree/usr/ vmcheck/usr/
 # ✀✀✀ BEGIN hack for https://github.com/projectatomic/rpm-ostree/pull/642 ✀✀✀
 ostree admin unlock || true
-for url in https://kojipkgs.fedoraproject.org//packages/ostree/2017.2/3.fc25/x86_64/ostree-{,libs-,grub2-}2017.2-3.fc25.x86_64.rpm; do
-    curl -sSL -O $url
-done
+ver=2017.3
```

Comment:
That also bumped the version from 2017.2 to 2017.3. Was that intentional?

Actually we should be able to dump this hack now. The `fedora/25/atomic` image has ostree-2017.2-3. (And it's nice, because I always have to cut ✂️ out that chunk on my computer because I use CAHC!).